### PR TITLE
[ko] Korean Translation for Lesson 6

### PR DESCRIPTION
**Pull request by WZNT-KimHyeJin**
### 6-1 : speech-recognition
- pi-microphone.ko.md

### 6-3 : spoken-feedback
- README.ko.md
- pi-to-text-speech.ko.md

### CODE REVIEW

File: 6-consumer/lessons/1-speech-recognition/translations/pi-microphone.ko.md [hunk 1]
```diff
@@ -1,140 +1,140 @@
...
-    def play_speech(speech):
-        with wave.open(speech, 'rb') as wave_file:
-            stream = audio.open(format=audio.get_format_from_width(wave_file.getsampwidth()),
-                                channels=wave_file.getnchannels(),
-                                rate=wave_file.getframerate(),
-                                output_device_index=speaker_card_number,
-                                output=True)
+   이 값을 적절한 카드 번호로 설정합니다. 키보드의 화살표 키를 사용하여 숫자로 이동한 다음 텍스트 파일을 편집할 때 일반적으로 새 숫자를 삭제하고 입력할 수 있습니다.
+   
+1. `Ctrl+x`를 눌러 변경 내용을 저장하고 파일을 닫습니다. `y`를 눌러 파일을 저장한 다음 `return`을 눌러 파일 이름을 선택합니다.
```
Comment (DobiIsFree) [inline 6-consumer/lessons/1-speech-recognition/translations/pi-microphone.ko.md:118]: 번호가 다 1. 로 되어있는게 맞나요??

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-actuator.kr.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# 야간 조명 만들기 - 라즈베리 파이
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-actuator.kr.md:1]: Didn't we decide to write Raspberry Pi in English?

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-actuator.kr.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+### 할 일 - 야간 조명 동작에 대한 프로그래밍을 해 봅시다.
+
+야간 조명에 대한 프로그래밍을 합니다.
+
+1. 라즈베리 파이 전원을 켜고 부팅될 때 까지 기다립니다.
+
+1. 이 할당의 이전 부분에서 만든 VS Code에서 야간 조명 프로젝트를 Pi에서 직접 실행하거나 원격 SSH 확장을 사용하여 연결합니다.
+
+1. 필요한 라이브러리를 가져오려면 아래 코드를 `app.py` 파일에 추가하십시오. 이는 다른`import` 줄 바로 아래 추가되어야 합니다..
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-actuator.kr.md:57]: two dots
..

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-sensor.kr.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+    이는 `GroveLightSensor` 클래스의 `light` 속성을 사용하여 0-1,023의 척도로 현재 빛의 밝기를 판독합니다. 이 속성은 핀에서 아날로그 값을 읽습니다. 이후 이 값이 콘솔에 출력됩니다.
+    
+1. 계속 밝기를 확인할 필요가 없으므로 `루프` 끝에 1초의 짧은 절전 시간을 추가한다. 절전 모드는 장치의 전력 소비를 줄여줍니다.
+
+    ```python
+    time.sleep(1)
+    ```
+
+1. VS Code의 터미널에서 아래 코드로 Python 앱을 실행 해 봅시다..
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-sensor.kr.md:77]: two dots
..

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-sensor.kr.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+    ```sh
+    python3 app.py
+    ```
+
+    밝기 값이 콘솔에 출력될 것이다. 광센서를 손으로 가려도 보면서 값이 어떻게 변하는지 확인 해 봅시다 :
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-sensor.kr.md:83]: Here, you used "-이다"

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-sensor.kr.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
+# 야간 조명 만들기 - 라즈베리 파이
```
Comment (jhYun505) [inline 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-sensor.kr.md:1]: 라즈베리 파이 -> Raspberry Pi

File: 6-consumer/lessons/3-spoken-feedback/translations/pi-text-to-speech.ko.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+                stream.write(data)
+                data = wave_file.readframes(4096)
+
+            stream.stop_stream()
+            stream.close()
+    ```
+
+    이 코드는 오디오를 캡처하는 것과 동일한 PyAudio 스트림을 사용합니다. 여기서 차이점은 오디오 데이터에서 데이터가 읽혀지고 출력 스트림으로 설정 된 스트림으로 푸시된다는 것입니다.
+    
+    샘플링 정도와 같은 스트림 세부 사항을 하드 코딩하는 대신 오디오 데이터로부터 읽힌다.
```
Comment (jhYun505) [inline 6-consumer/lessons/3-spoken-feedback/translations/pi-text-to-speech.ko.md:124]: Here "읽힌다" -> "읽힙니다"

**Review by jhYun505 — COMMENTED** [review]
Perfect except for small mistakes!

File: 6-consumer/lessons/1-speech-recognition/translations/pi-microphone.ko.md (hunk 1 above)
Comment (WZNT-KimHyeJin) [reply 6-consumer/lessons/1-speech-recognition/translations/pi-microphone.ko.md:118]: 엉엉.....T^T

File: 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-actuator.kr.md (hunk 2 above)
Comment (WZNT-KimHyeJin) [reply 1-getting-started/lessons/3-sensors-and-actuators/translations/pi-actuator.kr.md:1]: Oh... I miss that one... Thanks!

File: 6-consumer/lessons/1-speech-recognition/translations/pi-microphone.ko.md (hunk 1 above)
Comment (WZNT-KimHyeJin) [reply 6-consumer/lessons/1-speech-recognition/translations/pi-microphone.ko.md:118]: I think Markdown gives the order automatically.

**Comment by WZNT-KimHyeJin** [comment]
@jimbobbennett Please check this PR before you merge!!

**Comment by jimbobbennett** [comment]
@WZNT-KimHyeJin - can you fix the merge conflict please. I'm not sure what is the best resolution.

**Review by jimbobbennett — APPROVED** [review]

**Comment by WZNT-KimHyeJin** [comment]
@jimbobbennett The previous Korean translator used 'ko.md', so we also used 'ko.md'. I will modify other kr.md files to ko.md. 
But if 'kr' is the correct expression, We will modify all the files.

**Comment by WZNT-KimHyeJin** [comment]
@jimbobbennett I updated all kr files to ko. I'm sorry for the confusion and thank you for letting me know 🍀

**Pull request merged**